pldp/models/location.py

```diff
@@ -156,19 +72,97 @@ class Location(models.Model):
     subdivision = models.CharField(max_length=9,
                                    choices=SUBDIVISION_CHOICES,
                                    blank=True,
-                                   help_text=_("Line Geometry: indication of "
-                                               "whether the line is a "
-                                               "subdivision of a single survey"
-                                               " location. Area Geometry: "
-                                               "indication of whether an area "
-                                               "is a subdivision of a single "
-                                               "survey location."))
+                                   help_text=_("Indication of whether the "
+                                               "location is a subdivision of "
+                                               "a single survey location."))
     character = models.CharField(max_length=15,
                                  blank=True,
                                  choices=CHARACTER_CHOICES,
                                  help_text=_("Primary character of the survey "
                                              "location's immediate "
                                              "surroundings."))
+    geometry = models.GeometryField(help_text=_("Polygon or line that "
+                                                "describes the geometry "
+                                                "of the location. Polygons "
+                                                "are intended for counts of "
+                                                "people staying in an area "
+                                                "while lines are intended for"
+                                                "counts of people moving "
+                                                "across a threshold."))
+    type = models.CharField(max_length=4,
```

Comment:
As long as we're doing a major migration, do you think it'd be a heavy lift to change the name of this field to `geometry_type` so that we don't clash with the builtin function `type`?

Reply:
Great point! I'll make this change.